models/template.go

```diff
@@ -137,6 +137,12 @@ func PutTemplate(t *Template) error {
 	if err := t.Validate(); err != nil {
 		return err
 	}
+
+	record, err := GetGroupByName(t.Name, t.UserId)
```

Comment:
As mentioned before, let's remove these checks that aren't necessary for these changes.